src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -56,4 +59,4 @@ fragment BACKTICK: '`';
 QUOTED_IDENT: BACKTICK (('``') | ~('`'))+ BACKTICK;
 
 fragment DIGIT: [0-9];
-WS  :   [ \t\n\r]+ -> skip ;
+WS  :   [ \t\n\r]+ -> channel(HIDDEN) ;
```

Comment:
that was actually what I was expecting too, but they let you replace a token with a string.  In theory one could build a more limited rewriter that only allows deletion of tokens or adding tokens (not strings) that would be able to re-output a TokenStream